code/modules/projectiles/updated_projectiles/ammo_datums.dm

```diff
@@ -853,20 +854,21 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	accuracy_var_low = 7
 	accurate_range = 15
 	damage = 150
-	max_range = 20
+	max_range = 40
 
 /datum/ammo/rocket/wp/drop_nade(turf/T, radius = 3)
 	if(!T || !isturf(T))
 		return
 	playsound(T, 'sound/weapons/guns/fire/flamethrower2.ogg', 50, 1, 4)
+	explosion(T, -1, 1, 1, 1)
```

Comment:
this is a 3x3 heavy damage explosion, you should thats fair?

Reply:
its a 1x1 heavy damage explosion, then a 3x3 light explosion